avalanche/evaluation/metrics/confusion_matrix.py

```diff
@@ -243,8 +245,8 @@ def __init__(self,
                  num_classes: Union[int, Mapping[int, int]] = None,
                  normalize: Literal['true', 'pred', 'all'] = None,
                  save_image: bool = True,
-                 image_creator: Callable[[Tensor], Image] =
-                 default_cm_image_creator):
+                 image_creator: Callable[[Tensor, Sequence], Image] =
```

Comment:
Is it ok to make the assumption that `image_creator` expects the CM and the labels?
The issue is that in this `__innit__` we can't get the `class_order`, so we need to tell the image creator later what the labels are

Reply:
Yes, I think this is a reasonable assumption.